hw_2/script.js

```diff
@@ -0,0 +1,39 @@
+function randn_bm() {
+    let u = 0,
+        v = 0;
+    while (u === 0) u = Math.random();
+    while (v === 0) v = Math.random();
+    return (Math.sqrt(-2.0 * Math.log(u)) * Math.cos(2.0 * Math.PI * v) * 2) | 0;
+}
+
+function fillAndGroupRandom(count, randomFunc) {
+    const resultObject = {};
+    let wCount = count + 1;
+
+    while ((wCount -= 1)) {
+        const elem = randomFunc();
+        if (elem in resultObject) {
+            resultObject[elem] += 1;
+        } else {
+            resultObject[elem] = 1;
+        }
+    }
+    return resultObject;
+}
+
+function drawTable(obj) {
+    console.log(obj);
+    console.log("+---------+---------+");
+    console.log("|  Number | Count   |");
+    console.log("+---------+---------+");
+    Object.getOwnPropertyNames(obj)
+        .sort((x, y) => x - y)
```

Comment:
Do we need this sort function in our task?

Reply:
https://github.com/BlackTruth/javascript/blob/0f090223bfbe9aef482c5d350255c17995e68193/hw_2/script.js#L27-L28
Removed sorting object.